gradle/jacoco.gradle

```diff
@@ -1,14 +1,14 @@
 ext {
     // TODO: this should become 90% for INSTRUCTION and 0 for CLASS
-    AggregatedCoverageThresholds = ["INSTRUCTION": 0.852, "CLASS": 93]
+    AggregatedCoverageThresholds = ["INSTRUCTION": 0.847, "CLASS": 107]
     // TODO: each module should achieve 90% for INSTRUCTION and 0 for CLASS
```

Comment:
This was forgotten to be adjusted in [General: Add passkey login option #10656](https://github.com/ls1intum/Artemis/pull/10656) - therefore the coverage conditions are currently not met on develop; I will add coverage for the passkey feature in a follow up PR